src/Core/System/Snippet/SnippetFixer.php

```diff
@@ -15,7 +20,23 @@ public function __construct(private readonly SnippetFileHandler $snippetFileHand
     {
     }
 
-    public function fix(MissingSnippetCollection $missingSnippetCollection): void
+    /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Will get a second parameter `$invalidPluralization`
+     */
```

Comment:
sorry I missed that last time. Symfony uses the same mechanism, but already add the new parameter with a `@param` annotation. 
```suggestion
     * @param Snippets $invalidPluralization
     */
```

this shows users already, that there is a new parameter

Reply:
So you'd remove the deprecation?

Reply:
no no, as addition

Reply:
PHPstan doesn't like this. What would you prefer? Entry in phpstan-baseline.neon, removing this or do something like `@todo @param ...`?

Reply:
PHPStan does not recognize the parameter as it is commented, so we should ignore it in place

Reply:
Doesn't work. Even if I ignore without a rule, it still says "nothing to ignore". Since we don't do this anyway ins shopware at all, I'll go with `NOTE`. Searching the web a bit for it, I find that I can't ignore errors caused by phpstan comments itself. Since this PR takes unnecessarily long anyway, I'll stop the experiments here.

Reply:
Made obsolete, since I put this into a collection